### PR TITLE
made zoom link

### PR DESCRIPTION
**Pull request by nhudait**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -82,6 +82,10 @@ app.post('/webhook', (req, res) => {
 	}
 });
 
+app.get('/zoom', (req, res) => {
+  res.redirect('https://berkeley.zoom.us/j/98351933735?pwd=dE0yeEl5OEZyd05MOC9VbWxKOXUxQT09')
```
Comment (PotatoParser) [inline index.js:86]: Missing semicolon

**Review by PotatoParser — CHANGES_REQUESTED** [review]
Looks good! Just a small lint needed.

File: index.js (hunk 1 above)
Comment (nhudait) [reply index.js]: i think i added it

**Review by PotatoParser — APPROVED** [review]
Perfect!

**Pull request merged**